partcad/src/partcad/ai_ollama.py

```
@@ -115,12 +115,16 @@ def handle_content(content: AiContentFile):
 
         candidates = []
         for _ in range(options_num):
+            response = None
             retry = True
             while retry == True:
```

Ruff (0.15.21)

[error] Avoid equality comparisons to `True`; use `retry:` for truth checks

Replace with `retry`

(E712)